docs/data/base/components/number-input/QuantityInput.js

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react';
+import NumberInput from '@mui/base/NumberInput';
```

Comment:
Base UI is now in a beta phase. I think that it could be beneficial to prefix the components to be unstable, so we can fix bug freely.

```suggestion
import NumberInput from '@mui/base/Unstable_NumberInput';
```

For example, we have no lab version for Material UI or Joy UI version yet, doing the integration is likely to surface need for changes.

Reply:
Sounds good ~ what about related names like classnames, types, e.g.:

```diff
import NumberInput, {
-  numberInputClasses,
+  unstable_numberInputClasses,
-  NumberInputOwnerState,
+  Unstable_NumberInputOwnerState,
} from '@mui/base/Unstable_NumberInput';
```

Reply:
No preferences for them.

Reply:
We can follow what's done in the `Unstable_Grid` from the @mui/system.